src/main/cpp/t11_boxes.cpp

```diff
@@ -37,5 +37,26 @@
 using namespace std;
 
 int t11_boxes() {
-
-};
\ No newline at end of file
+int a1, b1, c1, a2, b2, c2;
+cin >> a1 >> b1 >> c1 >> a2 >> b2 >> c2;
+if (((a1==a2) && (b1==c2) && (c1==b2)) || 
+    ((a1==b2) && (b1==a2) && (c1==c2)) ||
+     ((a1==a2) && (b1==b2) && (c1==c2)) ||
+     ((a1==b2) && (b1==c2) && (c1==a2)) ||
+     ((a1==c2) && (b1==a2) && (c1==b2)) ||
+     ((a1==c2) && (b1==b2) && (c1==a2))) cout << "Boxes are equal";
+if (((a1<a2) && (b1<b2) && (c1<c2)) ||
+    ((a1<a2) && (b1<c2) && (c1<b2)) ||
+    ((a2<b2) && (b1<a2) && (c1<c2)) ||
+    ((a1<b2) && (b1<c2) && (c1<a2)) ||
+    ((a1<c2) && (b1<a2) && (c1<b2)) ||
+    ((a1<c2) && (b1<b2) && (c1<a2))) cout << "The first box is smaller than the second one";
+if (((a2<a1) && (b2<b1) && (c2<c2)) || 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Same expression on both sides of '<'.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4430725)